src/patches/dense_patch_computation.cpp

```diff
@@ -463,38 +444,20 @@ InstructionApplicationResult try_apply_instruction_direct_followup(
         {
             for(const auto& occupied_cell: busy_region->region.cells)
                 if(!slice.is_cell_free(occupied_cell.cell))
-                    return {std::make_unique<std::runtime_error>("Could not find free cell in BusyRegion"),
-                            {std::move(instruction)}};
```

Comment:
This move and the one below were broken by changing instruction to be a reference. Was causing one of the bugs in the dag pipeline folder